reporting/reports.py

```diff
@@ -3,7 +3,7 @@
 import re
 import sys
 
-from six import reraise
```

Comment:
Why is that?

Reply:
As told, the `spacewalk` codebase is heavily using `spacewalk-usix` for solving this particular situation with `with_traceback`. Sure we could use `six.reraise` but I found it simpler and cleaner to just keep tied to `spacewalk-usix` instead of adding an extra dependency.

IMHO getting rid of `spacewalk-usix` and migrate to `python-six` would be more like a tech debt.

Reply:
Noooooo.... 

The `usix` implements nothing but a quite shoddy way of reraise and some poor type mapping. It doesn't have `iteritems` it has nothing about unicode, strings, bytes etc. In a worst case, adding a proper `six` into `usix` package would be a compromise if you don't want to make sure `python-six` is there.

On SUSE Manager server in the scripts you can even just do:

```python
from salt.ext import six
```

It is there already anyway. And even proven working with 2.6, in case we need that. The only place when you cannot do it is traditional client. But this is still question if we don't want to migrate it.